packages/dapp/src/const/trackingKeys.ts

```diff
@@ -1,33 +1,108 @@
+/*
+  Name of an action that a user makes on the site. 
+  GA: in GA this is the name of the even that is fired
+*/
+export enum TrackingActions {
```

Comment:
By convention, we should use a singular type name for an enumeration unless its values are bit fields. 🙂

TrackingActions -> TrackingAction
TrackingCategories -> TrackingCategory
TrackingEventParameters -> TrackingEventParameter

Reply:
Good idea! It´s all renamed in latest commits